csfieldguide/chapters/content/en/big-data/sections/collecting-big-data.md

```diff
@@ -0,0 +1,130 @@
+If you are analysing the effectiveness of a treatment on women and your training data only includes Pākehā (NZ European) and European women, then should you automatically apply the same conclusions to Māori women?
+No, of course not, there are physiological differences between people of different ethnicities, and it would be inaccurate to generalise them into one group.
+This is why it is so important to make sure that you are capturing data that accurately portrays the group of people that you want to investigate.
+
+{interactive slug="data-bias" type="whole-page" text="true"}
+
+Data Bias Interactive
+
+{interactive end}
+
+It’s also important to think about how data sets captured in the past, or by other people, may have been conditioned by the biases of those people.
+The analysis of biased healthcare data has frequently disadvantaged groups of people in the past who were under-represented in data samples.
+
+## Store
+
+So now that all of this data has been collected, it needs to be stored, which is a challenge in itself.
+The first issue is simply the amount of data.
+As we mentioned earlier, Big Data often begins at terabytes, but more often than not, this expands to petabytes or even exabytes.
+This is where data centres come in.
+
+Data centres are warehouses (often enormous ones) containing many computers that are connected in a network. They are used to store and process large quantities of data. Data centres have become a popular way for organisations to solve the issue of not having enough computing power for the amount of data they collect. Companies such as Google, Amazon, Microsoft, and even Catalyst here in New Zealand offer services where other organisations can rent space in their data centres instead of having to build their own (an expensive endeavour).
+
+{image file-path="img/chapters/google-data-centre.png" caption="true" alt="Image shows a long row of server racks at a Google data centre in Oklahoma"}
+
+Server racks in the Google data centre in Oklahoma.
+
+{image end}
+
+These data centres store a mind-boggling amount of information. Just think, every video uploaded to YouTube, every tweet ever written, every purchase ever made on Amazon, and every comment on every blog on the internet exists somewhere in one of these data centres (in fact it will exist in several, as there will be multiple backups of it).
+
+{panel type="curiosity"}
+
+# A Data Centre on Punch Cards
+
+In 2013, xkcd creator Randall Munroe tried to answer the question [“If all digital data were stored on punch cards, how big would Google's data warehouse be?”](https://what-if.xkcd.com/63/) and with some clever internet-sleuthing calculated that over a quadrillion cards would be needed. It would be enough to cover all six states of New England in 4.5km of punch cards. And that was all the way back in 2013! Imagine how much bigger that pile would be now!
```

Comment:
I don't think many people outside of the US would understand New England, especially since it isn't a (single) state

Reply:
True, but we can't really change that in this example and it's a cool comic for illustrating the size of data in data centres. How about I write "New England, US" or "all of New England (six states in the US)"? 
It's pretty easy for people to look up New England, and we've got quite a few cool xkcd comics in the guide now.

Reply:
> It would be enough to cover all six states of New England in 4.5km of punch cards

> It would be enough to cover all six states of New England, US, in 4.5km of punch cards

> It would be enough to cover all of New England (six states in the US) in 4.5km of punch cards

I agree not to change the example, it's just unfortunate it's new **England**. I think the third is best. Also have we mentioned punch cards before anywhere?

Reply:
I don't think so, but they're pretty easy to just google